app/actors/DocumentationPollingActor.scala

```diff
@@ -33,7 +32,9 @@ object DocumentationPollingActor {
  * documentation.
  */
 class DocumentationPollingActor @Inject() (messages: MessagesApi, @Assisted repos: DocumentationGitRepos,
-                                           @Assisted documentationActor: ActorRef) extends Actor {
+                                           @Assisted documentationActor: ActorRef) extends Actor
+    with ActorLogging
+{
```

Comment:
Formatting is a bit off. Not a blocker, though.

Reply:
I dislike the "align at a random column" formatting but I also avoid pushing my own opinions on formatting in unrelated code.  So this is my attempt to mixing in a trait that would be too long for the same line.